frontend/device/remarkable/device.lua

```diff
@@ -146,6 +176,9 @@ function Remarkable1:suspend()
 end
 
 function Remarkable2:suspend()
+    -- Need to remove brcmfmac kernel module before suspend
+    turnOffWifi()
```

Comment:
`turnOnWifi(self.model)`

Reply:
"attempt to index local 'self' (a nil value)"
I think I get why that's happening: generic/device does
```
UIManager:scheduleIn(self.suspend_wait_timeout, self.suspend)
```

This is the first time I've ever touched Lua, so instead of trying to do some clever solution, I just hard-coded
```
turnOffWifi("reMarkable 2")
```
here, since it's inside the `Remarkable2` implementation anyhow.

Reply:
Bit difficult to check context on my phone, but to pass self along:
```lua
UIManager:scheduleIn(self.suspend_wait_timeout, self, self.suspend)
```

Reply:
Other way around (`self.suspend, self`), but yeah, I figured it was something like that.
I was hesitant to fiddle around in `generic` since I wasn't sure how hard it would be to trace through possible side-effects on other devices. But you've emboldened me :P
Both `scheduleIn` references to `self.suspend` updated.